mix.exs

```diff
@@ -69,7 +69,8 @@ defmodule Hexpm.MixProject do
       {:rollbax, "~> 0.5"},
       {:sweet_xml, "~> 0.5"},
       {:telemetry_poller, "~> 1.0"},
-      {:telemetry_metrics, "~> 0.6"}
+      {:telemetry_metrics, "~> 0.6"},
+      {:esbuild, "~> 0.2", only: :dev}
```

Comment:
Dependencies should be sorted alphabetically.

Reply:
It should be `{:esbuild, "~> 0.2", runtime: Mix.env() == :dev}` so that we can run esbuild when building for production also.

Reply:
Makes sense, Working on it